### ISSUE TITLE
"TypeError: label is null" error when adding a torrent from a new tracker that hasn't been added to the list on the sidebar yet

### ISSUE THREAD

**Issue by GH0STDATA** [Novik/ruTorrent#2586]
### Please complete the following tasks.

- [X] Web browser cache cleared
- [X] Link provided to install script if applicable
- [X] Not using broken rtinst install script
- [X] Web browser, ruTorrent, PHP and OS version provided

### Tell us about your environment

Web Browser: Firefox 119.0
ruTorrent: v4.2.6
PHP: PHP 8.1.23
OS: Ubuntu 22.04.3 LTS

### Tell us how you installed ruTorrent

https://github.com/crazy-max/docker-rtorrent-rutorrent

### Describe the bug

As the title of the post describes, when adding a torrent from a new tracker that isn't already on the rutorrent sidebar, a  JS type error occurs, and continues to occur every time you click or do an action on the page. It also looks like another bug happens in the GUI as well:

![firefox_vUMA6VajNJ](https://github.com/Novik/ruTorrent/assets/33645963/4e391b09-c4a7-4973-9b42-35668e1bf4a3)

The bug seems to go away once you refresh the page, but I figured I'd mention it here anyway.

### Steps to reproduce

1. Add a torrent from a tracker that isn't already listed in the sidebar.

### Expected behavior

The tracker should be added to the sidebar with no error.

**Comment by stickz** [comment]
I got a stack trace from someone else regarding this bug. Adding it here for reference.

```
TypeError: label is null from theWebUI.updateLabel@https://?.d.itsby.design/rutorrent/ line 2 > injectedScript:7668:13
updateLabels@https://?.d.itsby.design/rutorrent/js/webui.js?v=426:2355:10
theWebUI.updateLabels@https://?.d.itsby.design/rutorrent/ line 2 > injectedScript:7685:22
theWebUI.rebuildTrackersLabels@https://?.d.itsby.design/rutorrent/ line 2 > injectedScript:7769:7
theWebUI.addTrackers@https://?.d.itsby.design/rutorrent/ line 2 > injectedScript:7609:12
Ajax/<@https://???.d.itsby.design/rutorrent/js/rtorrent.js?v=426:1301:22
c@https://???.d.itsby.design/rutorrent/js/jquery.js?v=426:2:25266
fireWith@https://???.d.itsby.design/rutorrent/js/jquery.js?v=426:2:26015
l@https://???.d.itsby.design/rutorrent/js/jquery.js?v=426:2:77721
send/o/<@https://???.d.itsby.design/rutorrent/js/jquery.js?v=426:2:80204
webui.js:3186:14

**Comment by GH0STDATA** [comment]
Thank you for the quick hotfix! I apologize about not posting a stack trace. Next time I report a bug, I will remember to do that.

**Comment by stickz** [comment]
> Thank you for the quick hotfix! I apologize about not posting a stack trace. Next time I report a bug, I will remember to do that.

No problem. The fix from trimming fool is what resolves this problem properly. You can disable the track labels plugin until `v4.2.8` is released. This will remove the JavaScript error. This plugin is not required. The functionality is trivial for most people.

**Comment by GH0STDATA** [comment]
> No problem. The fix from trimming fool is what resolves this problem properly. You can disable the track labels plugin until `v4.2.8` is released. This will remove the JavaScript error. This plugin is not required. The functionality is trivial for most people.

Gotcha, good to know. You're right, I create my own labels to separate torrents based on the tracker anyway, so that plugin is pretty redundant. I've disabled the tracklabels plugin as you suggested.

**Comment by stickz** [comment]
Resolved in [v4.2.8](https://github.com/Novik/ruTorrent/releases/tag/v4.2.8).
